cordova-lib/src/plugman/registry/registry.js

```diff
@@ -364,6 +364,26 @@ function fetchPlugin(plugin, client, useNpmRegistry) {
     });
 }
 
+function processPluginVersion(plugin) {
```

Comment:
How much work would it be to make use of 'getLatestMatchingNpmVersion' ? that way we don't have to maintain two separate ways of doing the same thing (platforms vs plugins).